packages/flutter/test/material/text_field_test.dart

```diff
@@ -703,10 +703,10 @@ void main() {
 
     await expectLater(
       find.byType(TextField),
-      matchesGoldenFile('text_field_cursor_width_test.1.0.png', version: 0),
+      matchesGoldenFile('text_field_cursor_width_test.1.png'),
     );
     EditableText.debugDeterministicCursor = false;
-  }, skip: !isLinux);
```

Comment:
Looks like this was reintroduced when a change was accidentally merged to master.

Reply:
#49572